randovania/games/super_metroid/json_data/header.json

```diff
@@ -265,7 +265,7 @@
             },
             "IBJ": {
                 "long_name": "Vertical Infinite Bomb Jump",
-                "description": "By timing bombs precisely, the player can ride their blasts to gain an infinite amount of height.",
+                "description": "By timing bombs precisely, Samus can gan an arbitrary amount of height. ",
```

Comment:
*gain
Also please remove the whitespace between dot and quotation mark at the end

Reply:
Got the white space at the beginning of 'Gravity Jump too